src/components/modal/Title.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  title: string
+}
+
+const Title = ({ title }: Props) => {
+  return (
+    <div className="text-subtitle font-bold leading-subtitle tracking-[-2%] text-black">
```

Comment:
```suggestion
    <div className="subtitle-B text-black">
```

이미 존재하는 유틸리티 클래스를 활용하면 좋을 듯 합니다~

Reply:
감사합니다 초기에 짠 코드라 제가 실수를 했네용 ,,, ㅎㅎ 수정했습니다 ~